src/main/java/com/zufar/onlinestore/payment/service/PaymentCreator.java

```diff
@@ -0,0 +1,39 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.stripe.exception.StripeException;
+import com.stripe.model.PaymentIntent;
+import com.zufar.onlinestore.payment.converter.PaymentIntentConverter;
+import com.zufar.onlinestore.payment.dto.CreatePaymentDto;
+import com.zufar.onlinestore.payment.dto.PaymentDetailsWithTokenDto;
+import com.zufar.onlinestore.payment.entity.Payment;
+import com.zufar.onlinestore.payment.converter.PaymentConverter;
+import com.zufar.onlinestore.payment.repository.PaymentRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class PaymentCreator {
```

Comment:
add java doc please
it helps to understand this class for people who have no idea about stripe

Reply:
Done